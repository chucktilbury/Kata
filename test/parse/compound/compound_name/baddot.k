;-------------------------------------------------------------------------------
; Adding an extra dot in a name
;-------------------------------------------------------------------------------

var this.is.a..compound.name var_name
