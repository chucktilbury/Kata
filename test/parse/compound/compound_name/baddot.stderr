Syntax: /home/chuck/Src/Kata/test/parse/compound/compound_name/baddot.k: 5: 16: expected a SYMBOL but got .
Syntax: /home/chuck/Src/Kata/test/parse/compound/compound_name/baddot.k: 5: 16: expected a variable declaration but got .
Syntax: /home/chuck/Src/Kata/test/parse/compound/compound_name/baddot.k: 5: 16: expected module item or end of file but got var
Syntax: /home/chuck/Src/Kata/test/parse/compound/compound_name/baddot.k: 5: 16: module cannot be empty
